py_neuromodulation/nm_database.py

```diff
@@ -0,0 +1,100 @@
+import os
+import sqlite3
+from pathlib import Path
+import numpy as np
+import pandas as pd
+
+class NMDatabase:
+    """
+    Class to create a database and insert data into it.
+    Parameters
+    ----------
+    out_path_root : str
+        The root path to save the database.
+    folder_name : str
+        The folder name to save the database.
+    """
+    def __init__(self, out_path_root, folder_name, csv_path = None):
```

Comment:
Please add also typehints in the function calls for each parameter

Reply:
Good point, thanks. I added the typehints.